src/main/kotlin/team/themoment/gsmNetworking/domain/mentor/repository/TempMentorRepository.kt

```diff
@@ -0,0 +1,6 @@
+package team.themoment.gsmNetworking.domain.mentor.repository
+
+import org.springframework.data.repository.CrudRepository
+import team.themoment.gsmNetworking.domain.mentor.domain.TempMentor
+
+interface TempMentorRepository: CrudRepository<TempMentor, Int>
```

Comment:
저희는 `kotlin 코딩 컨벤션`을 사용하기 때문에 콜론 뒤에 한칸 띄어쓰기를 하셔야 합니다.
```suggestion
interface TempMentorRepository : CrudRepository<TempMentor, Int>
```

Reply:
수정하였습니다.
kotlin 코딩 컨벤션에 대해서 찾아보도록 하겠습니다